napari/components/_dims/tests/qtdims_tests.py

```diff
@@ -0,0 +1,69 @@
+# This is in interactive test
```

Comment:
This test currently does not run,

```python
python napari/components/_dims/tests/qtdims_tests.py
/Users/nicholassofroniew/anaconda3/lib/python3.6/site-packages/vispy/visuals/isocurve.py:22: UserWarning: VisPy is not yet compatible with matplotlib 2.2+
  warnings.warn("VisPy is not yet compatible with matplotlib 2.2+")
Traceback (most recent call last):
  File "napari/components/_dims/tests/qtdims_tests.py", line 39, in <module>
    widget = QtDims(dims)
  File "/Users/nicholassofroniew/Github/napari/napari/components/_dims/qtdims.py", line 55, in __init__
    self._set_num_sliders(dims.num_dimensions)
  File "/Users/nicholassofroniew/Github/napari/napari/components/_dims/qtdims.py", line 140, in _set_num_sliders
    self._create_sliders(new_number_of_sliders)
  File "/Users/nicholassofroniew/Github/napari/napari/components/_dims/qtdims.py", line 155, in _create_sliders
    slider = self._create_range_slider_widget(new_slider_axis)
  File "/Users/nicholassofroniew/Github/napari/napari/components/_dims/qtdims.py", line 197, in _create_range_slider_widget
    parent=self)
  File "/Users/nicholassofroniew/Github/napari/napari/_qt/range_slider/range_slider.py", line 286, in __init__
    QRangeSlider.__init__(self, slider_range, values, parent)
  File "/Users/nicholassofroniew/Github/napari/napari/_qt/range_slider/range_slider.py", line 46, in __init__
    self.setRange(slider_range)
  File "/Users/nicholassofroniew/Github/napari/napari/_qt/range_slider/range_slider.py", line 228, in setRange
    self.scale = self.end  - self.start
TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType'
```

The reason for this is that the `range` sent to the new slider is `(None, None, None)` which it doesn't know how to handle. It does a check for a `None` type but not a tuple of `None` types. Unclear how you want to fix